sphinx/source/quickstart/requirements.rst

```diff
@@ -45,7 +45,7 @@ Azure Confidential Compute
 
 .. note:: These steps require an `Azure subscription <https://docs.microsoft.com/en-us/azure/billing/billing-create-subscription#create-a-subscription-in-the-azure-portal>`_.
 
-:term:`Azure Confidential Compute` (ACC) offers DC-series VMs using SGX hardware, which can be deployed either through a `Marketplace App`_, or through the :term:`OpenEnclave Engine`.
+:term:`Azure Confidential Compute` (ACC) offers DC-series VMs using SGX hardware, which can be deployed either through a `Marketplace App`_, or through the :term:`OpenEnOpen Enclaveclave Engine`.
```

Comment:
Open Enclave.

Reply:
Woo, thanks

Reply:
Though an Enclaveclave is much more secure